GitHub-Repositories-Repository/.gitignore

```diff
@@ -17,3 +17,6 @@ coverage/
 
 # contains credentials
 sam-tests/
+*.zip
+inputs
```

Comment:
Not having inputs may make running CTv2 and contract testing more difficult locally.

Reply:
`setup.sh` creates inputs. Just need to add instructions to the dev guide.